### ISSUE TITLE
Error thrown when using simple regex like 

### ISSUE THREAD

**Issue by MichaelSu1983** [kaesetoast/find-in-files#9]
error is thrown when I ignore regex, or put simple regex like '.js$'. Why it doesn't work?

```
findInFiles.find("xxx", 'xxxx', '.js$')
    .then(function(results) {
        for (var result in results) {
            var res = results[result];
            console.log(
                'found "' + res.matches[0] + '" ' + res.count
                + ' times in "' + result + '"'
            );
        }
    });
```

```
`/Users/msu/test/node_modules/q/q.js:155
                throw e;
                ^

TypeError: Cannot read property 'match' of undefined
    at getResults (/Users/msu/test/node_modules/find-in-files/index.js:74:22)
    at /Users/msu/test/node_modules/find-in-files/index.js:92:30`
```

**Comment by caseprince** [comment]
When will this fix be made available on npm?

**Comment by kaesetoast** [comment]
@caseprince @MichaelSu1983 sorry for the delay. It is merged now and available as 0.5.0 on npm
